posthog/models/person/person.py

```diff
@@ -99,3 +99,24 @@ class Meta:
 
     # current version of the id, used to sync with ClickHouse and collapse rows correctly for new clickhouse table
     version: models.BigIntegerField = models.BigIntegerField(null=True, blank=True)
+
+
+class PersonOverride(models.Model):
+    """A model of persons to be overriden in merge or merge-like events."""
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(
+                fields=["old_person_id", "override_person_id"], name="unique override per old_person_id"
```

Comment:
shouldn't it be this (the team part isn't even really needed, but could help in the rare case of uuid collisions):
```suggestion
                fields=["team", "old_person_id"], name="unique override per old_person_id"
```

Reply:
I think both conditions achieve the same thing. I'll commit this :+1: